### DISCUSSION TITLE
Multi-monitor support

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#431] [UI&UX]
Not sure I need to justify it but: 

~More immersion
~Ability to keep map on central monitor and push reports, and other little windows, out of the way onto other monitor(s) (makes you feel like you are actually in a control room monitoring things). 

(side note: Stupid me... I went out and purchased a 40" TV last night to attach to my computer for just that very reason and was sadly disappointed when I found out that there was no multiple monitor support.)

**Comment by katepolak** [comment]
There is no need for a specific support for this, though there are some things that would work better with it. You can make an application span multiple monitors using either Nvidia Surround or AMD Eyefinity depending on your GPUs brand.

**Comment by JezzyBelle** [comment]
Unfortunately what you are proposing does not give me the options I am looking for. I am not looking for a larger game window with just a larger map to obscure with more little open windows. I tried that option and it just is just a larger version of the same thing. Those little windows (a truck I am monitoring.. a station I am watching, still share the same overall window) still cover the main game/map window.  
 I don't know if the game engine would support it, but I am betting many would love what I am requesting..

Yes, I know, you can hide them, etc. However, I would like to leave some open. 
Example: When I used to use Photoshop, you would have a canvas to work (main window) with tons of tools and menus.  They can obscure your canvas. When you have the monitor support I am referring to, you can drag those little windows onto another desktop. IE: one monitor could have the open game map unobscured.  The other monitor would have whatever additional windows you want left open.

I am just saying, if this support could be brought into the game it could be life changing! Imagine. Three monitors... Map in the middle... All my trucks I am concerned with listed to the left, and to the right, all the trains I am monitoring.
